metaflow/plugins/kfp/kfp.py

```diff
@@ -1092,8 +1092,10 @@ def call_build_kfp_dag():
             if self.notify:
                 with dsl.ExitHandler(self._create_exit_handler_op()):
                     call_build_kfp_dag()
+                    s3_sensor_op = create_s3_sensor_op()
```

Comment:
It seems you've caught a bug!
is there a regression test for this scenario? s3_sensor_op with exit notify?

Reply:
@talebzeghmi yup there is! The regression test is simply adding `--notify` to the integration tests for `s3_sensor`. I actually went ahead and added `--notify` to all the others tests as well to ensure we don't encounter issues like this in the future.